templates/webpack.config.js

```diff
@@ -18,7 +18,7 @@ const TerserPlugin = require('terser-webpack-plugin');
  * https://github.com/callstack/nativepack/blob/main/README.md
  *
  * The API documentation for the functions and plugins used in this file is available at:
- * https://react-native-webpack-toolkit.netlify.app/
+ * https://callstack-nativepack.netlify.app/
```

Comment:
can we get `nativepack.callstack.com` 😍 ?

Reply:
That's an improvement for later